### PR TITLE
# 11 replace welcome page with real homepage. Description : Make  appropriate changes to pastpaper hub. Changes:  Change title to Welcome to PastPaper hub, edit code to make it look better. 

### PR DESCRIPTION
**Pull request by Codebreake**

### CODE REVIEW

**Comment by powermukisa** [comment]
@Codebreake  add a better title and a description to this PR. the title should be a summary of what you done. The description should explain further. check example here https://docs.google.com/document/d/10LyidM2TZWy4n9bPDQHUwPN8xvtivFtUlVWNTUba3bk/edit

File: public/index.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Welcome to Firebase Hosting</title>
-
+    <title>Welcome to Firebase Hosting for PastPaper Hub</title>
```
Comment (powermukisa) [inline public/index.html]: Good job! Change this to "Welcome to PastPaper Hub"

File: public/index.html [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
+    <link href="PastPaperHub2.css" rel="stylesheet" type="text/css">
```
Comment (powermukisa) [inline public/index.html]: rename this to just "main.css"

File: public/index.html [hunk 3]
```diff
@@ -31,11 +31,62 @@
...
   <body>
-    <div id="message">
-      <h2>Welcome to</h2>
-      <h1>Past Paper Hub Test</h1>
-      <h1>by Rich Kitibwa</h1>
+    <div id="container">
+      <div class="header">
+          <div class= "Subhead">
+            <h1> <strong> PastPaper Hub </strong></h1>
+
```
Comment (powermukisa) [inline public/index.html]: no need for this extra space

File: public/index.html [hunk 4]
```diff
@@ -31,11 +31,62 @@
...
+                        <li>Machine language</li>
+                        <li> Bsiness computing</li>
+                    </ul>
+            
+             </li>
+          </ul>
+      </div>
+      
+      <div class="Mainbody">
+            <p> Preparing for exams and tests is one of the hard times most students go through while in school. PastPaper Hub was created bearing in mind the frustration of such  students ; giving  access to past papers and answers and also creating interactive discussions to enable students discuss some of the solutions.  </p>
```
Comment (powermukisa) [inline public/index.html]: realign  these so that they are more organised. I can do this for you if you like?

Comment (powermukisa) [reply public/index.html]: Also this line exceeds 120 characters. A line of code should not be that long. anything beyond 120 characters should go to the next line. check https://stackoverflow.com/a/29972073/2534091 on how to add a line to your Visual Studio IDE

Comment (Codebreake) [reply public/index.html]: Ok i will implement all the changes suggested

**Comment by Codebreake** [comment]
i formatted the code in the file . Hope it looks better now.

**Pull request merged**